src/schedule-tick.js

```diff
@@ -0,0 +1,9 @@
+import FlashMediaSource from './flash-media-source';
+
+const scheduleTick = function(func) {
```

Comment:
I think we have a bit of file-itis going on here :) Couldn't this go directly into flash-media-source.js?